packages/fiori/src/ViewSettingsDialog.js

```diff
@@ -0,0 +1,361 @@
+		 *
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @public
+		 */
+		 sortDescending: {
+			type: Boolean,
+		},
+		/**
+		 * Keeps recently focused list in order to focus it on next dialog open.
+		 *
+		 * @type {Object}
+		 * @private
+		 */
+		 _recentlyFocused: {
+			type: Object,
+
+		},
+		/**
+		 * Stores settings of the dialog before the first open.
+		 *
+		 * @type {Object}
+		 * @private
+		 */
+		 _initialSettings: {
+			type: Object,
+		},
+		/**
+		 * Stores settings of the dialog after confirmation.
+		 *
+		 * @type {Object}
+		 * @private
+		 */
+		 _confirmedSettings: {
+			type: Object,
+		},
+		/**
+		 * Stores current settings of the dialog.
+		 *
+		 * @type {Object}
+		 * @private
+		 */
+		 _currentSettings: {
+			type: Object,
+		},
+	},
+	slots: /** @lends  sap.ui.webcomponents.fiori.ViewSettingsDialog.prototype */ {
+		/**
+		 * Defines the <code>sorting</code> list.
+		 * @type {sap.ui.webcomponents.fiori.List}
+		 * @slot sorting
+		 * @public
+		 */
+		 "sorting": {
+			propertyName: "sorting",
+			type: HTMLElement,
+		},
+	},
+	events: /** @lends  sap.ui.webcomponents.fiori.ViewSettingsDialog.prototype */ {
+
+		/**
+		 * Fired when OK button is activated.
+		 *
+		 * @event sap.ui.webcomponents.fiori.ViewSettingsDialog#confirm
+		 * @param {String} sortOrder The current sort order selected.
+		 * @param {String} sortBy The current sort by selected.
+		 * @public
+		 */
+		"confirm": {
+			detail: {
+				sortOrder: { type: String },
+				sortBy: { type: String },
+			},
+		},
+		/**
+		 * Fired when Cancel button is activated.
+		 *
+		 * @event sap.ui.webcomponents.fiori.ViewSettingsDialog#cancel
+		 * @param {Object} settings The current settings.
+		 * @public
+		 */
+		"cancel": {},
+	},
+};
+
+/**
+ * @class
+ * <h3 class="comment-api-title">Overview</h3>
+ * The <code>ui5-view-settings-dialog</code> component consisting of several lists.
+ * One of them (<code>Sort
+ * order</code>) is built-in, and another (<code>Sort By</code>) must be provided by the developer.
+ * The selected options can be used to create sorters for the table.
+ *
+ * The <code>ui5-view-settings-dialog</code> interrupts the current app processing as it is the only focused UI element and
+ * the main screen is dimmed/blocked.
+ * The <code>ui5-view-settings-dialog</code> is modal, which means that user action is required before returning to the parent window is possible.
+ *
+ * <h3>Structure</h3>
+ * A <code>ui5-view-settings-dialog</code> consists of a header, content, and a footer for action buttons.
+ * The <code>ui5-view-settings-dialog</code> is usually displayed at the center of the screen.
+ *
+ * <h3>Responsive Behavior</h3>
+ * <code>ui5-view-settings-dialog</code> stretches on full screen on phones.
+ *
+ * <h3>ES6 Module Import</h3>
+ *
+ * <code>import "@ui5/webcomponents/dist/ViewSettingsDialog";</code>
+ *
+ * @constructor
+ * @author SAP SE
+ * @alias sap.ui.webcomponents.fiori.ViewSettigsDialog
```

Comment:
In order to copy the sample correctly, you should fix the typo here: sap.ui.webcomponents.fiori.ViewSetti**n**gsDialog

Reply:
done